tracer/src/Datadog.Trace/SamplingPriorityInternal.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="SamplingPriorityInternal.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace;
+
+internal static class SamplingPriorityInternal
```

Comment:
Something about `*Internal` always bugs me 😅 Is `SamplingPriorityValues` better?

Reply:
Funny, `SamplingPriorityValues` was my first choice and then I renamed it to `SamplingPriorityInternal`. I'll rename it back, thanks!

Reply:
Done in ebf9924b44578af2a09643a1ddbee2cda09f4f05.

(side note: `*Values` always makes me think of ["Addams Family Values"](https://www.imdb.com/title/tt0106220/))